roles/manage-aws-infra/tasks/update_dns.yml

```diff
@@ -20,6 +20,19 @@
     ttl: 300
   when: not ha_mode
 
+- name: Update Route53 with a CNAME to OCP master API
+  route53:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    zone: "{{ dns_domain }}."
+    record: "console.{{ env_id }}.{{ dns_domain }}."
```

Comment:
@makentenza we shouldn't hard code the values this way. Instead, we should allow the user to define `openshift_master_cluster_public_hostname` and `openshift_master_cluster_hostname` and use those names. Note that this will require us to move those variables out to all.yml instead of OSEv3.yml in our sample inventory.